src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js

```diff
@@ -255,4 +292,15 @@ const Index = ({
     </>);
 };
 
-export const SearchPage = withStyles(getStyles)(Index);
+export const SearchPage = ({ ...props }: Props) => {
```

Comment:
Not really huge fun of what is happening here but couldnt see a better way to do it. Let me know if you have ideas! :)